pyccl/cells.py

```diff
@@ -14,14 +14,14 @@ def angular_cl(
     ell,
     *,
     p_of_k_a=DEFAULT_POWER_SPECTRUM,
-    l_limber=-1,
+    ell_limber=-1,
```

Comment:
Just a super quick comment. We cannot do changes to the API without bumping the major version; i.e. without going to CCLv4. Since we don't want that, you need to ensure that the changes respect the API.

Reply:
Hmm but this isn't a new feature -- it's a fix

Reply:
@carlosggarcia done in 4e83abf54c13aea617f04f181aa8c6bdd278c479